sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -53,14 +53,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc
-  # Merge: 62e6d2f9 5fb82924
+  set(UNIFIED_RUNTIME_REPO "https://github.com/omarahmed1111/unified-runtime.git")
+
+  # commit be53fb3ba5bf1ac33051456d72c61b6a01d94a72
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Nov 8 13:32:46 2023 +0000
-  #     Merge pull request #1022 from 0x12CC/l0_usm_error_checking_2
-  #     [UR][L0] Propagate OOM errors from `USMAllocationMakeResident`
-  set(UNIFIED_RUNTIME_TAG ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc)
+  # Date:   Tue Oct 31 13:22:52 2023 +0000
+  # - Merge pull request #932 from Bensuo/cuda-cmd-buffers
+  # - [CUDA][EXP] CUDA adapter support for command buffers
+  set(UNIFIED_RUNTIME_TAG 7a95b06a762e2c4d1f4d2f76ba8cc1d2d0b09584)
```

Comment:
I'm assuming this will be updated with the revision from https://github.com/oneapi-src/unified-runtime/pull/1014 right?

Reply:
Yes, this is just a temporary part until the other PR is merged.